### DISCUSSION TITLE
WINDOWS版本，反馈几个影响使用的问题

### DISCUSSION THREAD

**Discussion by ZHUOLUERNV** [xfangfang/wiliwili#226] [General]
01.WINDOWS版本看不了自己关注的UP主，一点自己头像就显示注销账户，一点别人头像就显示取消关注，也就是进不去UP主的主页，这个真的特别影响使用。

02.希望窗口视频播放能实现左右鼠标点击调节亮度和音量的功能，现在点播放窗口只能倍速播放。希望能实现空格键暂停视频。

03.希望搜索框能直接搜索UP主，现在只能搜索关键字的视频，而且搜索框打开没有关闭键。

04.希望视频下方的弹幕开关和弹幕设置按钮换个颜色或样式，长得很像容易点错。

官方的PC客户端真的特别拉胯，大佬你做的PC客户端界面好看播放流畅，比官方强太多了，希望能一直维护下去，期待新版本上线！

**Comment by xfangfang** [comment]
1. 暂时没做这个功能，等以后有时间的，具体的更新会同步在：https://github.com/xfangfang/wiliwili/issues/64 ，可以订阅那个issue的通知。
2. 新增 F：全屏，空格：播放暂停，模仿手机端，鼠标左侧上下滑动调节亮度，右侧调节音量。（亮度调节目前在pc端仅限win笔记本或平板之类的）
3. 搜索up主功能依赖于前面第一个，如果第一个做了这个也会顺带着完成的；输入框快捷键做了微调，并且无论是否在设置里关闭提示都会强制提示（是的之前也有按键提示，不过你应该是在设置里关闭了）。
4. 弹幕在关闭时不显示设置按钮，避免误点。开启后因为颜色不同所以应该也不会误点。

上述这些功能可下载测试版体验（推荐windows下载最新出的带有d3d后缀的版本，性能更好），更多说明见：https://github.com/xfangfang/wiliwili/wiki
欢迎反馈。
